mbed_host_tests/host_tests_conn_proxy/conn_primitive_remote.py

```diff
@@ -124,7 +124,7 @@ def write(self, payload, log=False):
             self.selected_resource.write(payload)
             if log:
                 self.logger.prn_txd(payload)
-        return payload
+        return True
```

Comment:
I wasn't really sure for the remote primitive. In the past the `payload` was always returned from write, which is why I just replaced it with True. I think we could probably revisit the remote primitive when we start seeing more of it's use.